service/dap/server_test.go

```diff
@@ -3109,6 +3109,79 @@ func TestNextAndStep(t *testing.T) {
 	})
 }
 
+func TestNextInterrupted(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
+		t.Skip("test is not valid on ARM64")
```

Comment:
Why not? I understand the freebsd backend, but this is suspicious.

Reply:
Hmmmm I don't know where this came from... removing.